packages/open-vm-tools/open-vm-tools.spec

```diff
@@ -0,0 +1,106 @@
+%global buildver 17337674
+
+Name: %{_cross_os}open-vm-tools
+Version: 11.2.5
+Release: 1%{?dist}
+Summary: Tools for VMWare
+License: LGPL-2.1-or-later
+URL: https://github.com/vmware/open-vm-tools
+Source0: https://github.com/vmware/open-vm-tools/releases/download/stable-%{version}/open-vm-tools-%{version}-%{buildver}.tar.gz
+Source1: vmtoolsd.service
+Patch0001: 0001-no_cflags_werror.patch
+Patch0002: 0002-dont-force-cppflags.patch
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}libglib-devel
+BuildRequires: %{_cross_os}libtirpc-devel
+BuildRequires: %{_cross_os}libxcrypt-devel
+Requires: %{_cross_os}libglib
+Requires: %{_cross_os}libtirpc
+Requires: %{_cross_os}libxcrypt
+
+%description
+%{summary}.
+
+%package devel
+Summary: Files for development using the tools for VMWare
+Requires: %{name}
+
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n open-vm-tools-%{version}-%{buildver} -p1
+
+%build
+autoreconf -fi
+%cross_configure \
```

Comment:
nit: seems like `--disable-multimon` would also be pretty safe for our use case, given you have `--without-x` :)